config/locales/new/create_new_entry_in_editors_locale.de.yml

```diff
@@ -0,0 +1,20 @@
+de:
+  activerecord:
+    attributes:
+      pageflow/theming:
+        default_locale: Standard Sprache
+  pageflow:
+    admin:
+      themings:
+        default_locale_hint: Voreinstellung Sprache für Einträge
+    languages:
+      bg: Bulgarisch
+      cs-CZ: Tschechisch
+      fa-IR: Persisch
+      gn: Guarani
+      id: Indonesisch
+      ms: Malaysisch
+      nl-BE: Holländisch (Belgien)
+      pl-PL: Polnisch
+      pt-BR: Portugiesisch (Brasilien)
+      ru-RU: Russisch
```

Comment:
I'll leave these in. Who knows when they might be needed.

Reply:
If they are unused, I'd much prefer if we removed them and maybe stored the list somewhere else for later use.